_data/schemas/topics.yaml

```diff
@@ -53,7 +53,6 @@ properties:
     type: string
     minLength: 1   ## Anything not empty
     maxLength: 500 ## Arbitrary but suggest keeping it short in case we add the excerpts to an index at some point
-    pattern: "^\\*\\*"  ## Start excerpt with topic name in bold
```

Comment:
Would this work (rather than putting this check in the linter):

```
    pattern: "\\*\\*"  ## Excerpt must include topic name in bold
```

Reply:
Yeah.  Although I'd like to get the term mentioned as early in the excerpt as possible, using your suggestion is better than my crappy pipeline of regexes.  I have another change I want to make to the schemas; I'll bundle that in with it.  Thanks!